### PR TITLE
feat: update to work with newest unity-renderer

### PR DESCRIPTION
**Pull request by menduz**
- [x] Requires https://github.com/decentraland/unity-renderer/pull/689 to be merged

# What?
In order to be more gentle with browser's CPU, we would need to take advantage of network compression instead of the Unity JS fallback to uncompress the native builds. That is currently no possible, since the default behavior is to create `.unityweb` files compressed with a custom algorithm that only works in JS.

To make this sort of changes plug and play, we are moving the control of the selection of project files (`.unityweb`) to the https://github.com/decentraland/unity-renderer/pull/689 repo, isolating and encapsulating the loading behaviors in the target version.

Test link: https://play.decentraland.zone/branch/feat/new-renderer-initialization-interface/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/move-unity-configuration-to-renderer

More details at: https://github.com/decentraland/unity-renderer/pull/689

### CODE REVIEW

File: kernel/packages/unity-interface/loader.ts [hunk 1]
```diff
@@ -24,8 +18,54 @@ export type UnityGame = {
   SetFullscreen(): void
   Quit(): Promise<void>
 }
+// TODO: remove this after PR is merged and the UnityGame type migrates to @dcl/unity-renderer
```
Comment (kuruk-mm) [inline kernel/packages/unity-interface/loader.ts]: comment just as a reminder

**Review by kuruk-mm — APPROVED** [review]
Awesome! looks good!

File: kernel/packages/unity-interface/initializer.ts [hunk 2]
```diff
@@ -24,47 +24,28 @@ export type InitializeUnityResult = {
...
   preventUnityKeyboardLock()
 
   const canvas = document.createElement('canvas')
-  canvas.addEventListener('contextmenu', function (e) {
-    e.preventDefault()
-  })
+  // THIS IS COMMENTED BECAUSE Unity2020 already does this for us
+  // 1 canvas.addEventListener('contextmenu', function (e) {
+  // 1   e.preventDefault()
+  // 1 })
```
Comment (menduz) [inline kernel/packages/unity-interface/initializer.ts]:
```suggestion

```

File: kernel/package.json [hunk 3]
```diff
@@ -41,6 +41,7 @@
     "all": true
   },
   "devDependencies": {
+    "@dcl/unity-renderer": "^1.0.5913",
```
Comment (menduz) [inline kernel/package.json]:
```suggestion
```

**Pull request merged**